aspnetcore/getting-started/index.md

```diff
@@ -25,7 +25,7 @@ At the end, you'll have a working web app running on your local machine.
 
 ## Prerequisites
 
-* [.NET Core 2.2 SDK](https://www.microsoft.com/net/download/all)
+* [.NET Core 3.0 SDK](https://www.microsoft.com/net/download/all)
```

Comment:
This link seems to redirect to archives at https://dotnet.microsoft.com/download/archives.

Perhaps just :point_right: https://dotnet.microsoft.com/download

Reply:
@guardrex  nice catch. See #14483